packages/victory-shared-events/src/victory-shared-events.js

```diff
@@ -208,7 +208,7 @@ export default class VictorySharedEvents extends React.Component {
             name,
             baseProps,
             childEvents,
-            stringify(this.state[name])
+            stringify.bind(this, this.state[name])
```

Comment:
Wouldn't this cache a bound function rather than the stringified state object? Would something like this work instead?

```
const boundStringify = stringify.bind(this);
const sharedEventsCacheValues = [
  name,
  baseProps,
  childEvents,
  boundStringify(this.state[name])
];
```

Reply:
Ah, that's true! So the reason it seemed to work is just because it didn't work. 😄
Seems like the issue is that the recursive caching function enters an infinite loop. Made an update now which solves the problem by avoiding that block of code for `parent`. But of course I might be missing something or maybe the issue is a bit further up the chain.

Reply:
Apologies for the delay! This fix makes sense to me. I spent a bit of time looking checking for regressions, too, and all seems well. I'll get this merged for you and release it as a patch.